modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -2246,8 +2255,10 @@ private static synchronized InetAddress resetLocalHost() throws IOException {
         else {
             List<NetworkInterface> itfs = new ArrayList<>();
 
-            for (NetworkInterface itf : asIterable(NetworkInterface.getNetworkInterfaces()))
-                itfs.add(itf);
+            for (NetworkInterface itf : asIterable(NetworkInterface.getNetworkInterfaces())) {
+                if (!IGNORED_INTERFACES.contains(itf.getName()))
```

Comment:
Why we need to do this, if we use interface whose name starts with 'e' as localhost first?

Reply:
I think change in IngniteUtils should be also reviewed by core team.

Reply:
@alamar  Could you review this PR?